tools/tcat_ble_client/cli/base_commands.py

```diff
@@ -103,6 +105,38 @@ async def execute_default(self, args, context):
         return CommandResultTLV(tlv_response)
 
 
+class PingCommand(Command):
+
+    def get_help_string(self) -> str:
+        return 'Echo request to device.'
+
+    async def execute_default(self, args, context):
+        bless: BleStreamSecure = context['ble_sstream']
+        size = 10
+        max_payload = 512
```

Comment:
I'd suggest renaming these variables to make it more readable:
```suggestion
        payload_size = 10
        max_payload_size = 512
```

Reply:
fixed.